main.go

```diff
@@ -105,20 +110,20 @@ func main() {
 	// Re-enable pprof.
 	r.GET(*routePrefix+"/debug/pprof/*pprof", handlePprof)
 
-	log.Infof("Listening on %s.", *listenAddress)
+	level.Info(logger).Log("listen_address", *listenAddress)
 	l, err := net.Listen("tcp", *listenAddress)
 	if err != nil {
 		log.Fatal(err)
```

Comment:
This one is still using the old logging.

Reply:
Yeah, it looks like internally `log.Fatal` prints a message and then does an `os.Exit(1)` type thing so this will take a bit more refactoring